adsingestp/parsers/crossref.py

```diff
@@ -198,7 +201,11 @@ def _parse_contrib(self):
     def _parse_pubdate(self):
         pubdates_raw = self.record_meta.find_all("publication_date")
         for p in pubdates_raw:
-            datetype = p.get("media_type", "print")
+            try:
+                datetype = p.get("media_type", "print")
+            except Exception as err:
```

Comment:
This won't ever throw an exception, since the line above is assigning `"print"` as its backup. Also, we're mostly moving away from try/excepts in this library. I'd suggest something like this here instead:
```for p in pubdates_raw:
    if p.get("media_type"):
        datetype = p.get("media_type")
    else:
        logger.warn("...")
        datetype = "print"
```